### DISCUSSION TITLE
Issue with build from scratch

### DISCUSSION THREAD

**Discussion by mwakeham** [tailwindlabs/tailwindcss#6475] [Help]
This is very strange but stops any further CSS being applied. For some reason the build CSS gets the below inserted which stops the browsers rendering any further CSS. Any idea why?

.w-\[this-is\\\\\]w-\\\\\[weird-but-valid\] {
  width: this-is\\]w-\\[weird-but-valid;
}

**Comment by adamwathan** [comment]
Hey! The problem is your `content` configuration is matching `node_modules` — make sure you use a more specific configuration:

https://tailwindcss.com/docs/content-configuration#pattern-recommendations
